ambry-api/src/main/java/com.github.ambry/rest/RequestPath.java

```diff
@@ -91,9 +95,18 @@ public static RequestPath parse(String path, Map<String, Object> args, List<Stri
     // test if the last segment is a sub-resource
     RestUtils.SubResource subResource = null;
     int lastSlashOffset = path.lastIndexOf('/');
+    long blobSegmentNum = NO_BLOB_SEGMENT_IDX_SPECIFIED;
```

Comment:
Can you add some comments after line 95 providing an example of segment id? It's helpful  to understand the path format when segment id is included.

Reply:
done